core/src/messages/dns.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
+    nats::{SubscribeSubject, TypedMessage},
+    types::ClusterName,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum LocalDnsRecordType {
```

Comment:
What does `Local` refer to in this context? Can we just drop it and call this `DnsRecordType`?